Cargo.toml

```diff
@@ -240,6 +240,8 @@ rand = { version = "0.8.4", features = ["small_rng"] }
 form_urlencoded = "1.2.0"
 async-std = "1.12.0"
 web-time = "1.1.0"
+wgpu = "0.19"
+ouroboros = "*"
```

Comment:
can we move this to [dependencies] with an `optional=true` flag like `http` and then add `wgpu` to the required features list?